### DISCUSSION TITLE
Is this a UWP app or win32 app?

### DISCUSSION THREAD

**Discussion by oxygen-dioxide** [microsoft/winget-create#44] [Q&A]
Winget-create can "use all system resources", which shows that winget-create is a win32 app.
However, why does it need "access your internet connection" capability besides from using all system resources?
![image](https://user-images.githubusercontent.com/54425948/120191466-89322500-c24c-11eb-9118-4e8a98315f79.png)

**Comment by denelon** [comment]
@oxygen-dioxide I'm going to move this to a discussion. We will get an answer tomorrow when everyone is back from holiday.

**Reply by palenshus** [reply]
Tried removing the internetClient capability, the package still worked fine, looks like it was redundant. I'll go ahead and remove it
